test/test_history_saving.vader

```diff
@@ -19,6 +20,7 @@ Before:
   let g:ale_buffer_info = {}
   let g:ale_max_buffer_history_size = 20
   let g:ale_history_log_output = 0
+  let g:ale_cmd_wrapper = ''
```

Comment:
Add this in the `After` block too. That might be why the other test is failing.

Reply:
I did add that now. But, shouldn't the `Save` in `Before` and `Restore` in `After` do just that (I'm just guessing)?

Anyway, the error seems to be related to the `start` thing I found online, because it doesn't provide `exit_code`. I don't know how to handle that, I have no experience with Windows and none at hand.

Reply:
```
    (2/7) [EXECUTE] (X) ['command', 'job_id', 'status'] should be equal to ['command', 'exit_code', 'job_id', 'status']
```

Reply:
Any tips on how to move this thing forward? Unless someone knows a working `nice` equivalent for windows, my proposals would be:

* Don't test actual execution of the command on windows.
* Use some other dummy command there (eg. another layer of `cmd`?).

I believe this is the last thing blocking it, isn't it?

Reply:
I recommend using `if has('unix')` and skipping the tests on Windows.